src/main/java/Duke.java

```diff
@@ -1,10 +1,105 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static final String COMMAND_EXIT = "bye";
+    public static final String COMMAND_LIST = "list";
+    public static final String COMMAND_MARK_DONE = "done";
+    public static final String COMMAND_NEW_TODO = "todo";
+    public static final String COMMAND_NEW_DEADLINE = "deadline";
+    public static final String COMMAND_NEW_EVENT = "event";
+
+
     public static void main(String[] args) {
+        printStartMessage();
+        interactWithUser();
+    }
+
+    /**
+     * Prints the Duke application logo in the terminal.
+     */
+    public static void printAppLogo() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
     }
+
+    /**
+     * Prints a horizontal line in the terminal.
+     */
+    private static void printLine() {
+        System.out.println("  ──────────────────────────────");
+    }
+
+    /**
+     * Prints the application start message in the terminal.
+     */
+    public static void printStartMessage() {
+        printLine();
+        System.out.println("  Hello! I'm Duke\n  How may I assist you");
+        printLine();
+    }
+
+    /**
+     * Prints the application exit message in the terminal.
+     */
+    public static void printExitMessage() {
+        printLine();
+        System.out.println("  Goodbye! Hope to see you soon!");
+        printLine();
+    }
+
+    /**
+     * Prints the invalid command message in the terminal.
+     */
+    public static void printInvalidCommandMessage() {
```

Comment:
I think it is good that the method name follows the coding standard and is verbs and written in camelCase